tests/CounterTest.php

```diff
@@ -68,6 +68,10 @@ public function testShouldNotBeAbleToSetInvalidPeriod(): void
 
     public function testIncrementMustBeUniformAfterLimitIsReached(): void
     {
+        if (PHP_OS_FAMILY === 'Windows') {
+            $this->markTestSkipped('On Windows, the "usleep()" function used in this test may not work correctly.');
```

Comment:
Why?
Can you explain the reason in additional a comment or right in message?

Reply:
On Windows, the "usleep()" function used in tests may not work correctly: https://www.php.net/manual/en/function.usleep.php

This test on windows often is fail.

Reply:
https://github.com/yiisoft/rate-limiter/issues/11